StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1600UnitTests.cs

```diff
@@ -14,6 +16,26 @@
     public class SA1600UnitTests : DiagnosticVerifier
     {
         [Fact]
+        public async Task TestRegressionMethodGlobalNamespaceAsync()
+        {
+            // This test is a regression test for https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1416
+            var testCode = @"
+public void Test()
+{
+
+}";
+
+            var expected = new DiagnosticResult
+            {
+                Id = "CS0116",
+                Severity = DiagnosticSeverity.Error,
+                Locations = new[] {new DiagnosticResultLocation("Test0.cs", 2, 13)},
+                Message = "A namespace cannot directly contain members such as fields or methods"
```

Comment:
Not sure, but we have other tests where an English message is used. Have you run into problems with that?